index.html

```diff
@@ -12,7 +12,6 @@
   <main id="main-doc" class="main">
 
     <nav id="navbar">
-      <header> <h2> the technical doc. </h2> </header>
```

Comment:
I think it would be better to be careful with headers, many headers without a text tag('h1-h6')